galley/pkg/config/analysis/analyzers/testdata/service-no-port-name.yaml

```diff
@@ -0,0 +1,44 @@
+# No port name
```

Comment:
1. Please annotate the test cases here with comments indicating what should create messages (or not) and why.
2. Please also add a negative test case that shouldn't generate any messages.

Reply:
Done